src/common/modules/UnicodeFontHandler.js

```diff
@@ -162,18 +157,11 @@ function handle(info, tab) {
  * @returns {void}
  */
 function applySettings(unicodeFont) {
-	// Thunderbird
-	const menus = THUNDERBIRD ? browser.menus : browser.contextMenus;
+	const menus = browser.menus || browser.contextMenus; // fallback for Thunderbird
 
 	// menus.removeAll();
 
 	if (unicodeFont.changeCase) {
-		menus.create({
```

Comment:
Hmmm, why now remove the separator altogether?

Reply:
It was to separate the "Insert Emoji" item from the rest of the items (see screenshots [here](https://github.com/rugk/awesome-emoji-picker/issues/106#issuecomment-712144781)). I forgot to remove it when I updated my code for this PR and removed that item.